includes/class-sensei-course.php

```diff
@@ -126,8 +126,8 @@ public function __construct() {
 		// filter the course query when featured filter is applied
 		add_filter( 'pre_get_posts', array( __CLASS__, 'course_archive_featured_filter' ), 10, 1 );
 
-		// handle the order by title post submission
-		add_filter( 'pre_get_posts', array( __CLASS__, 'course_archive_order_by_title' ), 10, 1 );
+		// handle the order by title post submission.
+		add_filter( 'pre_get_posts', array( __CLASS__, 'course_archive_set_order_by' ), 10, 1 );
```

Comment:
Oops! I think I messed this one up.

Is it probable that I am now specifying the new ordering logic everywhere? Because previously this was only being done when the ordering query parameter was being sent and now it applies the default value no matter what.